CONFIGURATION.md

```diff
@@ -0,0 +1,95 @@
+# Configuration
+
+The Nuts Knooppunt application supports configuration through YAML files and environment variables.
+
+## Configuration Priority
+
+Configuration is loaded in the following order (later sources override earlier ones):
+
+1. Default values (hardcoded)
+2. YAML configuration file 
+3. Environment variables with `KNPT_` prefix
+
+## YAML Configuration
+
+The application automatically looks for configuration files in the following locations:
+
+- `config/knooppunt.yaml`
+- `config/knooppunt.yml`
+
+### Example YAML Configuration
+
+```yaml
+# mCSD (Mobile Care Services Discovery) configuration
+mcsd:
+  # Root directories to synchronize from
+  rootDirectories:
+    "example-org":
+      fhirBaseURL: "https://fhir.example.org/fhir"
+    "another-org":
+      fhirBaseURL: "https://fhir.another-org.com/fhir"
+  
+  # Local FHIR directory configuration
+  localDirectory:
+    fhirBaseURL: "http://localhost:8080/fhir"
+
+# mCSD Admin configuration
+mcsdadmin:
+  # Base URL for FHIR server used by admin interface
+  fhirBaseURL: "http://localhost:8080/fhir"
+
+# Nuts node configuration  
+nuts:
+  # Whether to enable the Nuts node component
+  enabled: true
+  
+  # Path to Nuts node configuration file (optional)
+  configFile: "config/nuts.yaml"
+```
+
+## Environment Variables
+
+Environment variables use the prefix `KNPT_` followed by the configuration path in uppercase with underscores:
+
+| Environment Variable | YAML Path | Description |
+|---------------------|-----------|-------------|
+| `KNPT_NUTS_ENABLED` | `nuts.enabled` | Enable/disable Nuts node component |
+| `KNPT_NUTS_CONFIGFILE` | `nuts.configFile` | Path to Nuts node configuration file |
+| `KNPT_MCSDADMIN_FHIRBASEURL` | `mcsdadmin.fhirBaseURL` | FHIR base URL for admin interface |
+| `KNPT_MCSD_LOCALDIRECTORY_FHIRBASEURL` | `mcsd.localDirectory.fhirBaseURL` | Local FHIR directory URL |
```

Comment:
- format this so that all columns are equal width
- change "Enable/disable Nuts node component " to "Enable embedded Nuts node"